.github/workflows/test.yml

```diff
@@ -41,40 +41,39 @@ jobs:
         dbimage:
           - mysql:5.7
           - mysql:8.0
-          - mariadb:10.4
-          - mariadb:10.11
+          - mariadb:10.2
+          - mariadb:11
+          - percona:5.7
           - percona:8.0
         swiftver:
-          - swift:5.6-focal
           - swift:5.7-jammy
           - swift:5.8-jammy
```

Comment:
Why not have have bionic/focal too? Doesn't hurt I guess.

Reply:
'cause truth is specific Ubuntu flavor matters to basically nothing any of Vapor's packages do and the rewrite branch is about to drop 5.6 support anyway (don't even talk to me about how tempted I am to drop 5.7 too >.<)